src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesString = sourceString
+    .split(';')
+    .filter(value => value.includes(':'))
+    .map(property => property.trim());
+
+  const callback = (allItems, item) => {
+    return {
+      ...allItems,
```

Comment:
I guess you could try `Object.assign` and mutate the `stylesObject` for better performance

Reply:
But it's not critical